conf/suse/master

```diff
@@ -537,18 +537,24 @@ syndic_user: salt
 # The renderer to use on the minions to render the state data
 #renderer: yaml_jinja
 
-# The Jinja renderer can strip extra carriage returns and whitespace
-# See http://jinja.pocoo.org/docs/api/#high-level-api
-#
-# If this is set to True the first newline after a Jinja block is removed
-# (block, not variable tag!). Defaults to False, corresponds to the Jinja
-# environment init variable "trim_blocks".
-#jinja_trim_blocks: False
-#
-# If this is set to True leading spaces and tabs are stripped from the start
-# of a line to a block. Defaults to False, corresponds to the Jinja
-# environment init variable "lstrip_blocks".
-#jinja_lstrip_blocks: False
+# TODO documentation
+# Ignoring documentation for now
+# (until agreement on implementation has been reached)
+
+# Default jinja environment options for everything but
+# salt sls files
+#jinja_env:
+#  line_statement_prefix: '#'
+#  line_comment_prefix: '##'
+#  trim_blocks: True
+#  lstrip_blocks: True
+
+# Jinja environment options for salt sls files
+#jinja_sls_env:
+#  line_statement_prefix: '#'
```

Comment:
@terminalmage for SLS files I would like to see the line_statemet_prefix enabled by default. I know you wrote few months ago - now way. But I still think it would be nice new "standard" that improves SLS readability.

I quite belive it should not affect old setups. But probably we should still check, test how frequent is (ie: %) in SLS of current formulas and that for example ``line_statement_prefix: '%'`` dont affect lines if it's not the first char on the line, etc...

Reply:
I agree, this would be nice to have enabled just for salt's sls files by default if possible. As far as my reading and testing has found there are extremely limited situations in yaml where `%` is allowed at the beginning of a line. But if extremely limited is still too much we could use something like `%%` instead, I've been unable to find any conflicts at all with `%%`.